hydroshare/settings.py

```diff
@@ -782,7 +782,7 @@ def _(s): return s # noqa
     import logging
 
     logging.disable(logging.CRITICAL)
-    DISABLE_HAYSTACK = True
+    # DISABLE_HAYSTACK = True
```

Comment:
@pkdash I know you disabled Haystack in order to speed up tests. Do you remember how much time we saved by doing that? We might have to revert and keep Haystack enabled for tests as Scott has here...

Reply:
@pkdash I think we will move forward with enabling Haystack even though it slows down tests, unless you object

Reply:
@devincowan If I remember correctly it probably saved about 10 min. I am OK with keeping the index on during test. In that case, we can remove the commented line 75.

Reply:
thanks @pkdash ...I'm not seeing a commented line 75 that is relevant. Did you mean in `settings.py` or some other file?

Reply:
thanks @pkdash ...I'm not seeing a commented line 75 that is relevant. Did you mean in `settings.py` or some other file?

Reply:
@devincowan Sorry the line is 785 not 75.